scripts/plugins/extauth-hook-AD.py

```diff
@@ -177,8 +177,21 @@ def _query_subject(self, sid):
             logger.error("subject %s from sid %s is not valid", subject, sid)
             return None, None
         # CONNAPP\test_user 1  # 1: user, 2: group
-        name, category = comps[0].strip(), comps[1].strip()
-        return name, True if category == "2" else False
+        is_group = True if comps[1].strip() =="2" else False
```

Comment:
I assume this can be simplified to `is_group = comps[1].strip() == "2"` (also added a space after `==`)

Reply:
ah, yes, my bad